### ISSUE TITLE
关于yolov11多卡训练导出pt文件报错

### ISSUE THREAD

**Issue by WXX-0827** [ultralytics/ultralytics#17575]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

我使用yolov11s并且在2张GPU下进行模型训练，再训练完成后导出pt文件时报错，但是这个pt文件却成功导出这是具体报错
![屏幕截图 2024-11-16 141111](https://github.com/user-attachments/assets/f9e840d1-59aa-4947-9416-8d9494661372)

**Comment by UltralyticsAssistant** [comment]
👋 Hello @WXX-0827, thank you for reaching out and sharing your question with us! This is an automated response to help you get started, and an Ultralytics engineer will also assist you soon 🚀. We recommend checking our [Docs](https://docs.ultralytics.com) where you can find useful resources and examples related to multi-GPU training and model export functionality.

If this is a 🐛 Bug Report regarding the export error, we would greatly appreciate it if you could provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). This will help us diagnose and resolve the issue more effectively.

For questions about custom training setups or results, please share more details, such as dataset examples and training logs, and ensure you’ve reviewed our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

You can engage with our community on different platforms:
- Join us for real-time discussions on [Discord](https://ultralytics.com/discord) 🎧.
- Participate in detailed discussions on [Discourse](https://community.ultralytics.com).
- Explore and contribute to threads on our [Subreddit](https://reddit.com/r/ultralytics).

## Upgrade

Ensure you are using the latest version of the `ultralytics` package, which includes all updated [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml). Please check that your Python environment is version **3.8** or later, and your PyTorch is **1.8** or newer:

```bash
pip install -U ultralytics
```

## Environments

Consider using one of these verified environments with preinstalled dependencies like [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/):

- **Notebooks** with free GPU support: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. Refer to the [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. Refer to the [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. Check the [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

As long as this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) tests are passing. These tests ensure the correct operation across all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you for your patience and collaboration!

**Comment by WXX-0827** [comment]
现在错误变成了下面图片的错误，但是我并没有修改任何代码
![屏幕截图 2024-11-16 222327](https://github.com/user-attachments/assets/32a98f35-978f-4f9f-ab3b-a89c7b24c54d)

**Comment by glenn-jocher** [comment]
@WXX-0827 the error appears to be related to DDP (Distributed Data Parallel) training cleanup. While the model export was successful despite the error message, this is likely a minor issue with the distributed training process termination. Since you're using multiple GPUs, you can safely ignore this error as it occurs during the cleanup phase after training is complete and doesn't affect the exported model's functionality. If you want to verify the exported model works correctly, try loading and running inference with it.

For more details on distributed training with YOLO, you can refer to the official documentation at https://docs.ultralytics.com/modes/train/#multi-gpu-training.

**Comment by Y-T-G** [comment]
You can try downgrading to 2.4.1

https://github.com/ultralytics/ultralytics/issues/17404#issuecomment-2463854126
